CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Pulumiverse
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing. These are just
+guidelines, not rules, so use your best judgment and feel free to propose
```

Comment:
judg`e`ment

Reply:
Tip: You can use code suggestions for these kinds of suggestions.

Example below:

```suggestion
guidelines, not rules, so use your best judgement and feel free to propose
```